internal/pikamon/bot.go

```diff
@@ -72,3 +75,20 @@ func (b *Bot) Start() error {
 func (b *Bot) Stop() error {
 	return b.discord.Close()
 }
+
+// Initialize bot cache object
+func createBotCache(cfg *Config) {
```

Comment:
The more idiomatic naming here is `newBotCache`, and it should return a pointer to the cache instead of setting a global.

Reply:
done, though this _may_ get removed in #24